cc1101/__init__.py

```diff
@@ -156,7 +156,11 @@ class CC1101:
     _PATABLE_LENGTH_BYTES = 8
 
     def __init__(
-        self, spi_bus: int = 0, spi_chip_select: int = 0, lock_spi_device: bool = False
+        self,
+        spi_bus: int = 0,
+        spi_chip_select: int = 0,
+        lock_spi_device: bool = False,
+        max_speed_hz: int = 55700,
```

Comment:
I would suggest renaming the kwarg from `max_speed_hz` to `spi_max_speed_hz`, so it is clear that the setting only affects the communication with the transceiver via SPI, but does not influence RF transmissions.
```suggestion
        spi_max_speed_hz: int = 55700,
```

Reply:
Agree, way better naming; I'll do.